src/Tester/GenericGrainTests.cs

```diff
@@ -287,6 +293,20 @@ public static void MyClassCleanup()
         }
 
         [TestMethod, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Generics")]
```

Comment:
Also, please mark the failing test with Ignore attribute. Otherwise, it will fail CI.